v21.2/data-domiciling-with-placement-restricted.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Data Domiciling with ALTER DATABASE ... PLACEMENT RESTRICTED
+summary: Learn how to implement data domiciling using CockroachDB's multi-region SQL capabilities and the ALTER DATABASE ... PLACEMENT RESTRICTED statement.
+toc: true
+---
+
+{% include {{page.version.version}}/sql/data-domiciling-intro.md %}
+
+## Overview
+
+This page has instructions for data domiciling in [multi-region clusters](multiregion-overview.html) using the [`ALTER DATABASE ... PLACEMENT RESTRICTED`](placement-restricted.html) statement. At a high level, this process involves:
+
+1. Controlling the placement of specific row or table data using regional or global tables - specifically, the [`REGIONAL BY ROW`](multiregion-overview.html#regional-by-row-tables), [`REGIONAL BY TABLE`](multiregion-overview.html#regional-tables), and [`GLOBAL`](multiregion-overview.html#global-tables) table localities.
+1. Further restricting where the data in those regional tables is stored using the [`ALTER DATABASE ... PLACEMENT RESTRICTED`](placement-restricted.html) statement, which constrains the replicas for a row or table to be stored in only the [home regions](set-locality.html#crdb_region) associated with those rows or tables.
```

Comment:
nit: I think the set locality portion doesn't exactly explain what a home region is either. Is there an appendix or something that specifies what exactly a home region is?